cluster-scope/overlays/ocp-staging/clusterroles/xdmod-permissions/clusterrole.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: xdmod-permissions
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
```

Comment:
`list` is the additional permission that was needed. the pervious rolebinding only had `get`